### PR TITLE
Added baking soda (don't ask)

### PR DESCRIPTION
**Pull request by the-og-gear**
:cl: MASMC
add: Baking soda and recipe (no current use, to be used in later PRs)
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)
Baking soda, to possibly allow more in-depth chef work, and to be used in future PRs as a valid reagent

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/food_reagents.dm [hunk 1]
```diff
@@ -632,6 +632,13 @@
 		M.adjust_nutrition(-3*nutriment_factor)
 	..()
 
+/datum/reagents/consumable/bakingsoda
+  name = "Baking Soda"
```
Comment (Kierany9) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm]: that's a funny lookin' tab you've got there

Comment (MASMC) [reply code/modules/reagents/chemistry/reagents/food_reagents.dm]: The funny part is that I only touched the stuff I wrote :v

Comment (MASMC) [reply code/modules/reagents/chemistry/reagents/food_reagents.dm]: Also 
image

Comment (MASMC) [reply code/modules/reagents/chemistry/reagents/food_reagents.dm]: Found the issue, resolving

**Comment by MASMC** [comment]
So on that topic, Atom decided "hey yeah your first line is going to be soft tabs, I'm putting in spaces"

**Comment by ike709** [comment]
nice going mancy

**Comment by MASMC** [comment]
Okay so I straight up used everything as it was used in the rest of the file, this should have compiled. It also compiled well on my machine. So...

File: code/modules/reagents/chemistry/reagents/food_reagents.dm [hunk 2]
```diff
@@ -632,6 +632,13 @@
 		M.adjust_nutrition(-3*nutriment_factor)
 	..()
 
+/datum/reagents/consumable/bakingsoda
```
Comment (Kierany9) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm]: shouldn't be plural

**Comment by MASMC** [comment]
Recipe conflict, please hold

**Comment by MASMC** [comment]
Okay so this is more of an in depth change than I thought, I'm just gonna close this and keep working :/

**Comment by MASMC** [comment]
There _will_ be recipe changes. And possibly many of them.

**Pull request closed without merging**